src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -49,6 +46,8 @@ private static void setupCommandTests(Application application) {
 		application.addTest("Run command", new SelectRunCurrentCommandOptionListener(DriverFeature.getDriverManager()));
 
 		application.addTest("Mouse figure", new SelectMouseFigureOptionListener(application.getFreePanel(), DriverFeature.getDriverManager()));
+
+		application.addTest("ICompoundCommandVisitorTest", new SelectICompoundCommandVisitorCounterListener());
```

Comment:
Please add the test that counts the 'currentCommand' (see Run command test impl.)